docker-compose.yml

```
@@ -0,0 +1,153 @@
+services:
+  vpn:
+    image: ${LOCAL_VPN_IMAGE}:${VERSION}
+    build:
+      context: .
+      dockerfile: Dockerfile.vpn
+    container_name: vpn
+    privileged: true
```

Locally, `make run-rp` fails for me as it fails to connect to the Hive cluster in my region. I think this is because the VPN is being instantiated in it's own Net namespace on the Linux kernel. Changing this to `hostNetwork: true` fixes the problem for me locally:

```suggestion
    privileged: true
    network_mode: host  # Use host network to set VPN for all containers running on the host
```

---

Updated VPN service configuration to use host network in `docker-compose.yml`